### ISSUE TITLE
An invalid or illegal string was specified

### ISSUE THREAD

**Issue by henrikheil** [LPology/Simple-Ajax-Uploader#126]
I'm not getting the standard example for single uploads to work.

The onSubmit-event is fired, but straight afterwards i'm getting the error message:
An invalid or illegal string was specified

This points to the code:
```javascript
if ( opts.encodeCustomHeaders && opts.customHeaders.hasOwnProperty( i ) ) {
    xhr.setRequestHeader( i, encodeURIComponent( headers[ i ] ) + '' );
} else {
    xhr.setRequestHeader( i, headers[ i ] + '' );
}
```

The Ajax Request doesn't get initiated at all.

Is this a known problem?

**Comment by LPology** [comment]
What is the file name of the file?

**Comment by LPology** [comment]
Fixed with version 2.3 Release. Closing out. Feel free to reopen or create another issue if you have any other issues.
